_episodes/01-basics.md

```diff
@@ -1,13 +1,20 @@
 ---
-layout: page
-title: Testing
-subtitle: Basics of Testing
-minutes: 5
+title: Basics of Testing
+teaching: 5
+exercises: 0
+questions:
+- "Why would you not want to test?"
+objectives:
+- "Understand the place of testing in a scientific workflow."
+- "Understand that testing has many forms."
+keypoints:
+- "Tests compare that the result observed from running code is the same as what was expected ahead of time."
+- "Tests should be written at the same time as the code they are testing is written."
+- "Assertions and exceptions are like alarm systems embedded in the software, guarding against exceptional bahavior."
+- "Unit tests try to test the smallest pieces of code possible, usually functions and methods."
```

Comment:
Were you meaning "Unit tests try to test the smallest pieces of code possible, usually functions and methods." as "Unit tests try to test the smallest atomic units of code possible, usually functions and methods."